packages/itwinui-react/src/core/Slider/Slider.tsx

```diff
@@ -446,6 +441,7 @@ export const Slider = React.forwardRef((props, ref) => {
           as='div'
           {...ticksProps}
           className={cx('iui-slider-ticks', ticksProps?.className)}
+          aria-hidden='true'
```

Comment:
this should probably not have `aria-hidden` because the user might have important text inside the ticks